rustls-mbedpki-provider/src/lib.rs

```diff
@@ -20,6 +20,21 @@ pub mod server_cert_verifier;
 pub use client_cert_verifier::MbedTlsClientCertVerifier;
 pub use server_cert_verifier::MbedTlsServerCertVerifier;
 
+/// A config about whether to check certificate validity period
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct CertActiveCheck {
+    /// If accept expired certificate, default to false
+    pub ignore_expired: bool,
+    /// If accept not active certificate, default to false
```

Comment:
Could be "Accept certificates that are not yet active"

Reply:
done